implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/Tuple.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace;
+
+class Tuple<X, Y> {
```

Comment:
Can we use `javafx.util.Pair` instead of creating new class?

Reply:
https://www.infoworld.com/article/3305073/java/removed-from-jdk-11-javafx-11-arrives-as-a-standalone-module.html - seems like javafx may not be a good choice

commons-lang3 has a Tuple class - would that dependency be ok

the advantage of my current approach is that it limits dependencies on outside classes

Reply:
I don't think we want to depend on anything just for a Tuple class. Ideally, we would eliminate the need for a Tuple class or we can keep a simple internal class for it.